### DISCUSSION TITLE
Feature Request: multi select on combobox / select

### DISCUSSION THREAD

**Discussion by dohomi** [rewindui/rewindui#58] [Ideas]
Hey very nice library would love to give it a go. Currently there isn't a multi-select option on either Select or Combobox component. Do you consider adding these options to the components?

**Comment by NickDunas** [comment]
Hello Dominic!

Yes definitely I do plan to add multi-select as I need it myself as well for my projects. 

I am thinking to add it as an option on the Combobox component and also create a new Tags component. I haven't decided yet what would be most appropriate and less confusing.

Any further thoughts or suggestions?

**Comment by dohomi** [comment]
@NickDunas I think the multi-select option inside Combobox in a same manner like MUI Autocomplete is very well known and probably the best bet.

**Comment by NickDunas** [comment]
Great, thank you! 

I will let you know once I release it.

**Comment by NickDunas** [comment]
Hello, @dohomi 

https://github.com/rewindui/rewindui/assets/33748867/70dec9a0-4f04-4f3a-9a10-8adabcfe3adc

Any thoughts on this before I publish it?

**Comment by NickDunas** [comment]
You can actually try it out [here](https://docs-65b6oklve-rewindui.vercel.app/components/combobox) if you want!
